operator/controllers/external_scaler.go

```diff
@@ -73,15 +77,17 @@ func createExternalScaler(
 	)
 	logger.Info("Creating external scaler Service", "Service", *scalerService)
 	servicesCl := cl.CoreV1().Services(appInfo.Namespace)
```

Comment:
@zroubalik Regarding your question [here](https://github.com/kedacore/http-add-on/pull/39#discussion_r570556306). This is the reconciler client that's passed on to the files, it's created by the manager on `main.go` so I think it's the same one you're talking about.

Reply:
What is the reason for creating a separate kubernetes clients (`K8sCl`,`K8sDynamicCl`)?  There's one already created for you: https://github.com/kedacore/http-add-on/blob/ffd8857da0dc355f3e2bcf9d70f5afdde31d7050/operator/main.go#L88

I can see that you are using all of the across the codebase, what is the specific reason for this?

Reply:
Oh I see, @arschles picked it up right now when we were pairing, he'll make the change

Reply:
https://github.com/khaosdoctor/http-add-on/pull/1